src/libYARP_OS/src/Network.cpp

```diff
@@ -140,6 +141,17 @@ static int noteDud(const Contact& src)
 # define CARRIER_DEBUG(fmt, ...)
 #endif
 
+yarp::os::ProtocolVersion::ProtocolVersion() :  protoMajor(YARP_PROTOCOL_VERSION_MAJOR),
+                                                protoMinor(YARP_PROTOCOL_VERSION_MINOR),
+                                                protoPatch(YARP_PROTOCOL_VERSION_PATCH)
+{}
+
+yarp::os::ProtocolVersion::ProtocolVersion(int Major, int Minor, int Patch) :
```

Comment:
Hi @barbalberto , in the function declaration (.h) the three parameters Major,Minor,Patch are `const`, however in the function definition (.cpp) they are not. Is it intended?